docs/_posts/2020-08-22-ingest-multiple-tables-using-hudi.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Ingest multiple tables using Hudi"
+excerpt: "Ingesting multiple tables using Hudi at a single go is now possible. This blog gives a detailed explanation of how to achieve the same using `HoodieMultiTableDeltaStreamer.java`"
+author: pratyaksh.sharma.hudi
+category: blog
+---
+
+When building a change data capture pipeline for already existing or newly created relational databases, one of the most common problems that one faces is simplifying the onboarding process for multiple tables. Ingesting multiple tables to Hudi dataset at a single go is now possible using `HoodieMultiTableDeltaStreamer` class which is a wrapper on top of the more popular `HoodieDeltaStreamer` class. Currently `HoodieMultiTableDeltaStreamer` supports **COPY_ON_WRITE** storage type only and the ingestion is done in a **sequential** way.
```

Comment:
my 2 cents. take your call though. Instead of "one of the most common problems that one faces is simplifying the onboarding process for multiple tables", how about "one of the major challenges is the complexity in onboarding multiple tables".

Reply:
Guess will keep it in its original form. :) Thank you for your kind suggestion though.